### PR TITLE
Fix python importing

### PR DESCRIPTION
**Pull request by mpage21**

### CODE REVIEW

File: static/downloadables/runPrsCLI.sh [hunk 1]
```diff
@@ -410,7 +413,7 @@ calculatePRS () {
                     exit 1
                 elif ! [[ $(echo $filename | tr '[:upper:]' '[:lower:]') =~ .vcf$|.txt$ ]]; then
                     # check if the file is a valid zipped file (check getZippedFileExtension for more details)
-                    zipExtension=`$pyVer -c "import grep_file; grep_file.getZippedFileExtension('$filename', True)"`
+                    zipExtension=`$pyVer $SCRIPT_DIR/grep_file.py "zip" "$filename" "True"` # TODO THIS NEEDS TO BE TESTED!!!!
```
Comment (MattCloward) [inline static/downloadables/runPrsCLI.sh:416]: Don't forget to test this.

File: static/downloadables/parse_associations.py [hunk 2]
```diff
@@ -538,3 +538,8 @@ def runParsingAndCalculations(inputFilePath, fileHash, requiredParamsHash, super
     if num_processes is None or (type(num_processes) is int and num_processes > 0):
         with Pool(processes=num_processes) as pool:
             pool.map(parseAndCalculateFiles, paramOpts)
+
+
+if __name__ == "__main__":
+    runParsingAndCalculations(sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4], sys.argv[5], sys.argv[6], sys.argv[7], sys.argv[8], sys.argv[9], sys.argv[10], sys.argv[11], sys.argv[12], sys.argv[13])
```
Comment (MattCloward) [inline static/downloadables/parse_associations.py:544]: Perhaps do "from sys import argv" like you did with the other files to keep them consistent.

**Review by MattCloward — APPROVED** [review]
This looks great! See my two comments.

File: static/downloadables/parse_associations.py (hunk 2 above)
Comment (mpage21) [reply static/downloadables/parse_associations.py:544]: sys was already imported, so I decided to just use that

**Pull request merged**